packages/jest-runtime/tsconfig.json

```diff
@@ -17,6 +17,7 @@
     {"path": "../jest-snapshot"},
     {"path": "../jest-source-map"},
     {"path": "../jest-test-result"},
+    {"path": "../jest-transform"},
```

Comment:
this was missing, leading to some weird inconsistencies in type error reporting 🤷‍♂